packages/camera/ios/Classes/CameraPlugin.m

```diff
@@ -816,6 +816,8 @@ - (void)handleMethodCallAsync:(FlutterMethodCall *)call result:(FlutterResult)re
         }];
       }
       result(reply);
+    } else {
+      result(FlutterMethodNotImplemented);
```

Comment:
I think what you had before would have fallen through to `FlutterMethodNotImplemented`, whatever you prefer though.

Reply:
Yeah but it'd make the other `else` blocks not run on iOS 10+ :\